test/load/allocation/README.md

```diff
@@ -0,0 +1,10 @@
+This test creates a proper amount of GameServers at the initial step, switch them to Allocated state and finally shut them down (`automaticShutdownDelayMin` flag in a simple-udp).
+
+1) Run kubectl apply -f ./fleet.yaml
+2) Run `runAllocation.sh` script to perform this test. You can provide a number of runs as a parameter (3 is a default value)
+
+Number of allocated GameServers = numberOfClients * reqPerClient (see `allocationload.go`)
+
+To run this test in under the normal conditions number of replicas in the yaml file should be >= allocated GameServers.
```

Comment:
Also not sure what this means either. Can we rephrase? Is this inline with what you meant?
```suggestion
If this test is running without error, the number of replicas specified in the fleet.yaml (defaults to 100), should always be greater than the number of Allocated GameServers
```

I am not sure how there could be more Allocated GameServers than those in the Fleet? Maybe I missed the point of the sentence?

Reply:
Yes, the idea was to say "Number of Request to Allocate GameServers" from the above formula.